testing/fuchsia/run_tests.sh

```diff
@@ -40,11 +41,13 @@ reboot() {
 
   ./fuchsia_ctl -d $device_name ssh \
       -c "log_listener --dump_logs yes" \
+      --timeout-seconds $ssh_timeout_seconds \
       --identity-file $pkey
 
   echo "$(date) START:REBOOT ------------------------------------------"
   # note: this will set an exit code of 255, which we can ignore.
   ./fuchsia_ctl -d $device_name ssh -c "dm reboot-recovery" \
+      --timeout-seconds $ssh_timeout_seconds \
```

Comment:
We may want to use the default here, this operation should complete within milliseconds.